### PR TITLE
Created portfolio page

### PR DESCRIPTION
**Pull request by bhushan354**
- Changed spacing and other issues in my header.
- Used only one h1 tag this time and wrote the other thing using h2 tag as mentioned by reviewer.
- Added proper background image as said.
- Also used linters to check any errors.

### CODE REVIEW

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] Your design currently lacks a background image as shown below, this is an important part which should be in the main section of your portfolio website, please kindly add the background image using css
- [ ] There is too much spacing between the top bar and the main title, please reduce the amount of spacing, see the figma template for reference :+1: 
- figma design
f-2
- your design
f-2-s

File: README.md [hunk 1]
```diff
@@ -1 +1,188 @@
...
+
+# 📖 [My Portfolio] <a name="Basic HTML and css Page"></a>
+
+> This is basic my portfolio page I build it with HTML and CSS
+
+
+## 🛠 Built With <a name="built-with"></a>
+1. Linters
+2. HTML
+3. CSS
```
Comment (Efiamotu-1) [inline README.md:47]: - [OPTIONAL]
These three highlighted parts are not needed as they currently look like repetition, the tech stack covers completely and is descriptive enough to tell anyone that these were the technologies used to build the application :+1:

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @bhushan354 , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vickymarz — APPROVED** [review]
Hi @bhushan354, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested changes implemented ✅ 



## Optional suggestions

- Kindly ensure that your linters are passing before requesting for a review. This PR should have been invalidated by the previous reviewer because your linters are not showing the green checkmark

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**